src/App.js

```diff
@@ -19,9 +25,12 @@ import { useEffect } from "react";
 import SignupAlertModal from "./Components/UI/Modal/SignupAlertModal";
 import AlertModal from "./Components/UI/Modal/AlertModal";
 import NotFound from "./Pages/NotFound/NotFound";
+import AnimateLoader from "./Components/Animation/AnimateLoader";
+import AnimateNote from "./Components/Animation/AnimateNote";
 
 function App() {
   const { showLogin, showSignup, showSignupAlert, showError } = useModal();
```

Comment:
You can use Barrel exports to simplify imports here and make them more readable.
Check this out for more info: https://medium.com/@klauskpm/do-a-barrel-export-aa5b79b76b05

Reply:
Noted, will fix this